layouts/partials/tw/donate-prefill-lg.html

```diff
@@ -0,0 +1,184 @@
+{{ $hed := .hed | default "Become a Member" }}
+{{ $cta := .cta | default "Donate Now" }}
+
+{{ $btnClass := `outline-none min-w-[180px] inline-block rounded py-3 px-6 text-white font-sans font-black text-lg tracking-wide ring ring-transparent` }}
+{{ $activeClass := `bg-red-6 text-white hover:bg-red-5 focus:ring-red-7 active:ring-red-9` }}
+{{ $inactiveClass := `bg-s-2 text-s-9 hover:bg-s-3 focus:ring-s-7 active:ring-s-9` }}
+{{ $radioClass  :=  `rounded-full border-s-1 appearance-none mr-4 order-first w-6 h-6 checked:bg-black shadow-[0_0_0_3px_rgba(255,255,255,1)]`}}
+{{ $contributeTheme := `leaders`}}
+<div
+  data-ga-category="donate-prefill-lg"
+  class="no-js:hidden mx-auto w-full max-w-screen-md "
+  x-data="{
+    active: '{{ $activeClass }}',
+    inactive: '{{ $inactiveClass }}',
+    buildClass(cs, val) {
+      return Object.fromEntries(cs.split(/ /).map(c=>[c, val]));
+    },
+    init() {
+      this.activeClasses = {
+        ...this.buildClass(this.active, true),
+        ...this.buildClass(this.inactive, false),
+      }
+      this.inactiveClasses = {
+        ...this.buildClass(this.active, false),
+        ...this.buildClass(this.inactive, true),
+      }
+    },
+    amount: '84',
+    freq: 'm',
+    showOther: false,
+    btnClass(cond) {
+      return cond? this.activeClasses : this.inactiveClasses;
+    },
+    setAmt(freq, amount) {
+      this.freq = freq;
+      this.amount = amount;
+      this.showOther = false;
+    },
+    get donateURL() {
+      let campaign = this.$el.
+        closest('[data-sf-campaign]')?.dataset.sfCampaign ||
```

Comment:
data-sf-campaign is a Salesforce ID that reports to Salesforce where on the site a donation came from. As a final step, we need to ask Joanna what ID she wants for these pages and then just add it to the top of the page so it trickles down to this component.

Reply:
I've captured this in my pre-launch checklist